frontend/src/pages/Home.tsx

```diff
@@ -5,12 +5,30 @@ import ProjectList from "./ProjectList";
 const Dashboard: React.FC = () => {
     return (
         <>
-            <h3>Home</h3>
-            <p>
-                This is the alfa version of AcidWatch.
-                <br />
-                Create a new project to organize and share your simulation runs.
-            </p>
+            <div style={{ textAlign: "left", margin: "20px" }}>
+                <h1>Welcome to AcidWatch</h1>
+                <h2>Your Gateway to Understanding CO2 Impurities</h2>
+                <p>
+                    AcidWatch is a tool designed to empower researchers, chemists, and industry professionals by providing a single interface to access many models, each capable of calculating and/or predicting chemical reactions involving impurities in CO2 streams.
+                </p>
+                <p>
+                    Our goal is to democratise and open up the discussion around CO2 impurities, fostering collaboration and innovation in the field.
+                </p>
+            </div>
+
+            <div style={{ textAlign: "left", margin: "20px" }}>
+                <h3>Start Using AcidWatch Today</h3>
+                <p>
+                    Run your simulation in the "Models" tab on your right or explore available experiment results in the "Lab results" tab.
```

Comment:
is it on the right?

Reply:
oohhh, my bad 😅 It is a wonder I have a driving license, right? 😆